### ISSUE TITLE
Document thumbprint verification step during appliance initialization

### ISSUE THREAD

**Issue by stuclem** [vmware/vic-product#1940]
Per https://github.com/vmware/vic-product/issues/637. This adds an extra step in https://vmware.github.io/vic-product/assets/files/html/1.4/vic_vsphere_admin/deploy_vic_appliance.html.

Uncertain whether this is in 1.4.3 or 1.5.0. Issue https://github.com/vmware/vic-product/issues/637 is marked as being in 1.5.0, but is included in the UI automation Epic https://github.com/vmware/vic-product/issues/1432, which is being done in 1.4.3. TBC.

**Comment by stuclem** [comment]
For this one, we need a build of the 1.4.3 OVA. Not sure whether we have one of those yet. Check with @lgayatri.

**Comment by renmaosheng** [comment]
@wjun to share one build once his code is merged into 1.4.3

**Comment by vidya-v** [comment]
Hi @lgayatri, can you please let us know where I can get  a build of the 1.4.3 OVA?

**Comment by stuclem** [comment]
@vidya-v normally the latest builds are posted here: https://console.cloud.google.com/storage/browser/vic-product-ova-builds (you need to keep on scrolling down until all the entries have loaded). However, I'm not yet seeing any 1.4.3 builds.

**Comment by lgayatri** [comment]
@vidya-v , we will receive 1.4.3 build on 8/29.

**Comment by jitinkumar2018** [comment]
@lgayatri @vidya-v @stuclem @renmaosheng 
While trying to obtain the thumbprints of certificates of  the VIC appliance file server and management portal with the following commands:
1)openssl x509 -in  /opt/vmware/fileserver/cert/server.crt -noout -sha1 -fingerprint
2)openssl x509 -in  /data/admiral/cert/server.crt -noout -sha1 -fingerprint
I hit an error saying the files don't exist in both cases. ( After manual registration of the VIC appliance via UI )

**Comment by stuclem** [comment]
@jitinkumar2018. I just checked with a 1.4.3 appliance and the paths to the certs have indeed changed.

I found the admiral certs in the place you describe. However, the folder  `/storage/data/fileserver` appears to be empty. 

There is also a folder `/storage/data/certs` that contains certificate files. @renmaosheng what certificates are these? Are these the fileserver certs?
